gnomad/sample_qc/ancestry.py

```diff
@@ -100,6 +99,38 @@ def assign_population_pcs(
     :return: Hail Table or Pandas Dataframe (depending on input) containing sample IDs and imputed population labels, trained random forest model
     """
     from sklearn.ensemble import RandomForestClassifier
+
+    def expand_pd_array_col(
```

Comment:
I considered this more of a util but I don't have any example of where else it would be used in current pipelines so moving it here works for me.

Reply:
yeah, I don't think we've ever used it for anything else. Also we really don't have much pandas tools so I think that nesting it under the sole function that uses it is the right move!

Reply:
If this isn't using anything from the outer function's scope, then unnesting it would make both functions a bit easier to read. If it's not intended to be public, adding a leading underscore to the function name marks it private / undocumented.

Reply:
well, this is really just a small helper for this function and it's unrelated to the rest of the module, I think nesting it is best

Reply:
Ok. Generally though, nesting a function makes it more difficult to understand because you can't tell whether or not the nested function uses the outer function's parameters or other variables in the outer function's scope. If it's not nested, then it's easy to see what the inputs are.

Reply:
All right, I inlined this. Since we don't need all the extra params and all it only adds 1 line of code to the main function body and I'm pretty sure will make everyone happy :)

Reply:
Excellent, 39 lines of code deleted in 4d1a31e. That looks like the best solution.